### PR TITLE
Dev/pick and place

### PR DESCRIPTION
**Pull request by Ruthrash**
Hi all in this PR we wish to review two new features added:

- record grasps of arbitrary objects using AR tag (easily switch with any pose estimator)
       1. See script [here](https://github.com/pairlab/franka_arm_infra/blob/dev/pick_and_place/robot_toolkit/test_grasp_recorder.py)
       2. Instructions for running script [here](https://github.com/pairlab/franka_arm_infra/tree/dev/pick_and_place/robot_toolkit/docs)

- pick and place of the arbitrary object for which we have recorded grasps 
       1.  See script [here](https://github.com/pairlab/franka_arm_infra/blob/dev/pick_and_place/robot_toolkit/test_pick_n_place.py)      
       2.  Instructions for running script [here](https://github.com/pairlab/franka_arm_infra/tree/dev/pick_and_place/robot_toolkit/docs)

### CODE REVIEW

**Comment by Ruthrash** [comment]
https://github.com/pairlab/franka_arm_infra/assets/19574538/aae6f26d-9106-46f4-8f69-3176aa72bc01

video showing grasp recorder functionality

**Comment by Ruthrash** [comment]
https://github.com/pairlab/franka_arm_infra/assets/19574538/07ba9e34-7629-4a7e-84e6-1277ba07cac6

video showing pick and place functionality

File: robot_toolkit/test_grasp_recorder.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import numpy as np
+from robot_arm_algos.src.pick_and_place.object import Object
+from robot_arm_algos.src.tags_detection.aruco_board import ArucoBoard, ArucoBoardData
+from robot_arm_algos.src.camera.realsense_camera import RealSenseCamera
+from robot_arm_algos.src.pick_and_place.record_grasps import RecordGrasps
+from robot_arm_algos.src.robot_camera_calibration.data_collector.zmq_robot_pose_collector import ZmqRobotPoseCollector
+test_config = {
+    "aruco_board_data":  ArucoBoardData(dictionary="DICT_6X6_1000",
```
Comment (sagniksom) [inline robot_toolkit/test_grasp_recorder.py:8]: can this be swapped for a generic pose estimator? Like april tags?

**Comment by Ruthrash** [comment]
This PR is outdated and most uptodate PR is #35

**Pull request closed without merging**